src/response_logic.py

```diff
@@ -39,15 +43,30 @@ async def create_response(text: str, name: str, chat_id: int) -> str:
                 return f"Your account was already inactive."
 
         elif first_word == "send" or first_word == "/send":
-            memory = select_random_memory(user)
-            if memory is None:
-                return (
-                    "The user is not in the system, please join by typing; *join* or /join"
-                )
-            elif memory is False:
-                return "No memory found in the system. Please add memory, for further information you can type *help* or /help"
+            if len(split_text) == 1:
+                memory = select_random_memory(user)
+                if memory is None:
+                    return (
+                        "The user is not in the system, please join by typing; *join* or /join"
+                    )
+                elif memory is False:
+                    return "No memory found in the system. Please add memory, for further information you can type *help* or /help"
+                else:
+                    return memory.reminder
             else:
-                return memory.reminder
+                try:
+                    number_of_sending = int(split_text[1])
+                    if not (1 <= number_of_sending < 50):
+                        return "Please give a number which is 1<n<50  after *send*"
+                except:
+                    return "Please give a number which is 1<n<50  after *send*"
+
+                all_memories = list_memories(user)
+                number_of_sending = min(len(all_memories), number_of_sending)
+                selected_memories = random.sample(all_memories, number_of_sending)
```

Comment:
I think this random sampling method is inefficient. Think about someone who has tons of memories to remind. We can use the random row selection functionality of SQL.

Reply:
You may me right, instead of fetching all memories of a user, we can fetch the desired amount randomly.
However with the ORM structure when a user is fetched, all of its Reminders(Memories) are fetched as well.
Let's create this as an issue as a potential future problem.